scripts/run_e2e_tests.py

```diff
@@ -269,6 +269,7 @@ def run_tests(args):
             app_yaml_path,
             port=GOOGLE_APP_ENGINE_PORT,
             log_level=args.server_log_level,
+            automatic_restart=False,
```

Comment:
Should we add a comment on top explaining e2e test doesn't need to restart the server because we are not expecting to do code changes while the e2e test is running?